scrapers/AlexLegend/AlexLegend.yml

```diff
@@ -0,0 +1,48 @@
+name: AlexLegend
+
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - alexlegend.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[contains(concat(" ", normalize-space(@class), " "), " content-title ")]/h1

```

Comment:
normalize space is unnecessary

Reply:
That is a fundamentally different selector that does not accommodate modern dynamic styles of web pages. Something as simple a new style class would be likely to break the scraper which is the purpose of the contains normalize-space pattern.